common/crypto/attestation-api/test/CMakeLists.txt

```diff
@@ -23,12 +29,15 @@ ADD_EXECUTABLE(${VERIFY_EVIDENCE_APP}
     common/test-utils.cpp
     )
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${SGX_COMMON_CFLAGS}")
+
```

Comment:
I guess you added it so logging gets SGX_BUILD?  Wouldn't it be better/more robust to do that inside `${LOGGING_PATH}/CMakeVariables.txt` so it doesn't accidentally gets omitted in other new cmakefiles consuming logging eventually? Obviously might lead to duplicate definitions in some cases but that should not be a real problem?